SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -18,7 +18,16 @@ internal class LanguagePatcher
         private static readonly Dictionary<string, Dictionary<string, string>> originalCustomLines = new Dictionary<string, Dictionary<string, string>>();
         private static readonly Dictionary<string, string> customLines = new Dictionary<string, string>();
 
-        internal static void Postfix(ref Language __instance)
+        internal static void RepatchCheck(ref Language __instance, string key)
+        {
+            if (string.IsNullOrEmpty(key) || !customLines.TryGetValue(key, out _))
```

Comment:
Use `ContainsKey` instead of TryGetValue with discard

```suggestion
            if (string.IsNullOrEmpty(key) || !customLines.ContainsKey(key))
```